src/splitInteger.test.js

```diff
@@ -4,16 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-  expect(splitInteger(17, 4))
-    .toHaveLength(4);
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
+});
+
+test(`should return a part equals to a value
+  when splitting into 1 part`, () => {
+  expect(splitInteger(15, 1)).toEqual([15]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-  expect(splitInteger(32, 6))
-    .toEqual([5, 5, 5, 5, 6, 6]);
+  expect(splitInteger(20, 5)).toEqual([4, 4, 4, 4, 4]);
```

Comment:
The test description does not match the test implementation. The description talks about sorting parts in ascending order if they are not equal, but the test checks for the specific case of 20 divided into 5 parts, where all parts are equal.

Reply:
> The test description does not match the test implementation. The description talks about sorting parts in ascending order if they are not equal, but the test checks for the specific case of 20 divided into 5 parts, where all parts are equal.

Sorry, already fixed my mistakes